lib/Runtime/Types/SimpleTypeHandler.cpp

```diff
@@ -1,4 +1,4 @@
-//-------------------------------------------------------------------------------------------------------
+\//-------------------------------------------------------------------------------------------------------
```

Comment:
In case it hasn’t be noticed yet I believe that the \ on this line is the source of the xplat compile failures.

Reply:
You're probably right. I'm just now getting back to this one, so I'll fix that. Thanks.